utils/s3/client.hh

```diff
@@ -90,6 +90,10 @@ struct stats {
     std::time_t last_modified;
 };
 
+struct filler_exception final : std::runtime_error {
```

Comment:
This change does deserve a regression test

Reply:
added